assets/dev/scss/frontend/widgets/divider.scss

```diff
@@ -137,12 +139,16 @@
 	direction: rtl;
 }
 
+.e-con {
```

Comment:
This will cause a problem, because it will apply to all `.e-con` elements. We had that issue earlier on.

Reply:
Removed